lib/solidus_retail/order/generate_pos_order.rb

```diff
@@ -45,17 +45,37 @@ def create_order
 
```

Comment:
I'd like to follow some kind of convention instead of having the code at two different places. 

I'd like to hear your suggestions about how we can have a middle ground here. 
Here are mines:
* One suggestion would be that I could move non-rails related stuff to the lib folder
* Another would be that you move your stuff into the app folder